helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -32,6 +32,9 @@ imagePullPolicy: "IfNotPresent"
 iscsi:
   chapSecretName: ""
 
+  # Disable the pre-install hook used to validate the chap secret. This can be useful if you are using Argo CD
+  disablePreInstallHook: false
```

Comment:
This should be in the global scope, not in `.iscsi.disablePreInstallHook` as we might use the pre-install hooks for other other things.

Rename to `.disablePreInstallHooks` (with an `s`).